src/rest/main/main.go

```diff
@@ -56,6 +56,7 @@ func init() {
 	flag.StringVar(&caFile, "cacert", "", "CA certificate for client certificate validation")
 	flag.StringVar(&cliCAFile, "clicacert", "", "CA certificate for CLI client validation")
 	flag.Var(clientAuth, "client_auth", "Client auth mode(s) - <none,cert,jwt,password|user(deprecated)> default: password,jwt")
+	flag.BoolVar(&server.RbacDisable, "rbac-disable", false, "Disable RBAC")
```

Comment:
Would be good to have the separator be an underscore instead of a dash, to keep it consistent with `client_auth`. So this should be `rbac_disable`. 

Can we also mention that it's enabled by default? "Disable RBAC (default: false)" or something.

Reply:
changed